### PR TITLE
Fixing KNE textproto for Nokia

### PR DESCRIPTION
**Pull request by MarcCharlebois**
Changed interface key nomenclature, removed public image link for tagged srlinux:latest

This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia’s intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind.

### CODE REVIEW

File: topologies/kne/nokia/srlinux/dut.textproto [hunk 1]
```diff
@@ -4,7 +4,7 @@ nodes: {
     vendor: NOKIA
     model: "ixr10"
     config: {
-        image: "ghcr.io/nokia/srlinux:latest"
```
Comment (alexmasi) [inline topologies/kne/nokia/srlinux/dut.textproto:7]: Any reason for switching this value? It was originally targeting the public image by default to make it as easy as possible for users to run tests (without having to load in a custom image into their k8 cluster)

The interface name changes look good to me

Comment (MarcCharlebois) [reply topologies/kne/nokia/srlinux/dut.textproto:7]: The main purpose for FP is testing OC compliance. OC is only available for specific chassis types that requires a license, which means that in *most* cases people would have a custom image loaded.

If someone wants to run the public image in a k8s cluster, I would suggest just the textproto from https://github.com/openconfig/kne/tree/main/examples/nokia/srlinux-services as it's more of a generic use case.

Comment (alexmasi) [reply topologies/kne/nokia/srlinux/dut.textproto:7]: Okay thats fine with me, can you also update https://github.com/openconfig/featureprofiles/blob/131970e41bb2a5359e8e943f43c32ead408a05d0/cloudbuild/virtual.sh#L114 for the CI trigger?

Comment (MarcCharlebois) [reply topologies/kne/nokia/srlinux/dut.textproto:7]: No problem, I can package it in this PR, just to confirm that it looks right

Change from: 
` sed -i "s/ghcr.io\/nokia\/srlinux:latest/us-west1-docker.pkg.dev\/gep-kne\/nokia\/srlinux:ga/g" /tmp/kne/"${kne_topology}" `

to: 
`sed -i "s/srlinux:latest/us-west1-docker.pkg.dev\/gep-kne\/nokia\/srlinux:ga/g" /tmp/kne/"${kne_topology}"`

**Review by alexmasi — APPROVED** [review]

**Pull request merged**